sources/pmwebapi.go

```diff
@@ -110,7 +110,44 @@ func typeLabel(units string, pcpType string, name string) string {
 	} else {
 		name += unit
 	}
-	name = strings.Replace(name, "count / sec", "count_per_second", -1)
+	return name
+}
+
+func fixNaming(name string) string {
+	name = strings.ToLower(name)
+	switch name {
+	case "seconds":
+
+	case "milliseconds":
+
+	case "nanoseconds":
+
+	case "nanosec":
+		name = strings.Replace(name, "nanosec", "nanoseconds", -1)
+	case "millisec":
+		name = strings.Replace(name, "millisec", "milliseconds", -1)
+	case "count / sec":
+		name = strings.Replace(name, "count / sec", "count_per_second", -1)
+	case "sec":
+		name = strings.Replace(name, "sec", "seconds", -1)
+	case "mbyte":
+		name = strings.Replace(name, "mbyte", "megabytes", -1)
+	case "kbyte":
+		name = strings.Replace(name, "kbyte", "kilobytes", -1)
+	case "kilobytes":
+
+	case "megabytes":
+
+	case "bytes":
+
+	case "bytes_byte":
+		name = strings.Replace(name, "bytes_byte", "bytes", -1)
+	case "byte":
+		name = strings.Replace(name, "byte", "bytes", -1)
+	default:
+		name = strings.Replace(name, "failcnt", "failcount", -1)
```

Comment:
Why is your default case to replace `failcnt` with `failcount`. I like to interpret the default case as the situation that doesn't follow special parsing or rules. So, the `failcnt` should probably a case statement of its own, and you probably don't even need a default in this scenario because all other instances stay the same.

Reply:
I got it, went ahead and changed this